lib/rubocop/cop/rspec/before_after_all.rb

```diff
@@ -0,0 +1,42 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Check that before/after(:all) isn't being used.
+      # This leads to flaky tests when being split out.
+      #
+      # @example
+      #   # bad
+      #   describe MyClass do
+      #     before(:all) { do_something }
+      #     after(:all) { do_something_else }
+      #   end
+      #
+      #   # good
+      #   describe MyClass do
+      #     before(:each) { do_something }
+      #     after(:each) { do_something_else }
+      #   end
+      class BeforeAfterAll < Cop
+        MESSAGE = 'Avoid the use of before/after(:all) to avoid specs flakyness when split out'.freeze
```

Comment:
'flakiness' seems to be the more common spelling. (500,000 vs 8000 matches on Google)